secrets-manager.tf

```diff
@@ -0,0 +1,15 @@
+data aws_secretsmanager_secret "uri-host" {
```

Comment:
we have to pass secret name from outside as variable (sure you can have some default) instead of hardcodding,
and also one single secret is far enough for preserving all the values, we do not need multiple ones